source/extensions/all_extensions.bzl

```diff
@@ -14,3 +14,21 @@ def envoy_all_extensions(blacklist = []):
 
     # These extensions can be removed on a site specific basis.
     return [v for k, v in all_extensions.items() if not k in blacklist]
+
+# Core extensions needed to run Envoy's integration tests.
+_core_extensions = [
+    "envoy.access_loggers.file",
+    "envoy.filters.http.router",
+    "envoy.filters.http.health_check",
+    "envoy.filters.network.http_connection_manager",
+    "envoy.filters.network.redis_proxy",
```

Comment:
This is used in the config for the hot_restart test. I'm unsure if it is needed by the test, so perhaps this target can be removed.

Reply:
+1 let's remove it.

Reply:
done